src/magento1/harness.yml

```diff
@@ -89,6 +89,9 @@ attributes:
       steps:
         - run n98-magerun.phar config:set 'web/unsecure/base_url' "https://${APP_HOST}/"
         - run n98-magerun.phar config:set 'web/secure/base_url'   "https://${APP_HOST}/"
+        - run n98-magerun.phar sys:setup:incremental -n
+        - run n98-magerun.phar index:reindex:all
+        - run n98-magerun.phar cache:clean
```

Comment:
would `- app migrate` be too odd alternative?

Reply:
I was thinking projects may override the migrate steps to do something they don't want to happen on init, but I doubt that will happen. Will update!

Reply:
Updated